python/lsst/obs/base/read_defects.py

```diff
@@ -0,0 +1,40 @@
+from lsst.meas.algorithms import Defects
+import os
+import glob
+from dateutil import parser
+
+
+def read_defects_one_chip(root, chip_name, camera):
+    files = glob.glob(os.path.join(root, chip_name.lower(), '*.ecsv'))
+    parts = os.path.split(root)
+    instrument = os.path.split(parts[0])[1]  # convention is that these reside at <instrument>/defects
+    defect_dict = {}
+    for f in files:
+        date_str = os.path.splitext(os.path.basename(f))[0]
+        valid_start = parser.parse(date_str)
```

Comment:
This would be a bit easier to read as `dateutil.parser.parse(date_str)`. I had to go hunting to figure out what parser was and where it came from.

Reply:
Done.